src/main/java/dukeApp/files/DukeException.java

```diff
@@ -0,0 +1,37 @@
+package dukeApp.files;
+
+public class DukeException extends Throwable {
+    protected String task;
+
+    /**
+     * Returns an error message if the command does not exist
+     *
+     * @param task command
+     * @return command error message
+     */
+    public String errorMsg(String task) {
+        if (task.equals("todo") || task.equals("deadline") || task.equals("event")) {
+            return "☹ OOPS!!! The description of a " +task+ " cannot be empty.";
+        }
```

Comment:
perhaps you could increase the bounds of this to include inputs such as "todo " or "deadline "?